src/http.c

```diff
@@ -36,7 +36,7 @@
  * Fixed size of the buffer for outgoing HTTP requests.
  * Initial size of the buffer for incoming HTTP responses.
  */
-#define HTTP_BUFFER_SIZE 0x8000
+#define HTTP_BUFFER_SIZE 0x10000
```

Comment:
From 5574b45:
> Increase the HTTP buffer capacity when needed. You never know.
> The previous size of 32 KB used to work well from 2015 to 2020.
> In 2020 a case was reported where 32 KB were not enough anymore
> and we increased the buffer size to 64 KB. Someday 64 KB might
> not be enough either. Yet in most cases 32 KB are more than
> enough. So start with 32 KB and increase well beyond 64 KB if
> needed, instead of bailing out with ERR_HTTP_SSL.

Thank you for catching this. The comment in 5574b45 seems to imply that it **increased** the buffer size from 32 KB (`0x8000`) to 64 KB (`0x10000`), instead it was **decreased** the other way round!!

Here are all the previous changes:
* 5574b45 on 2020-04-15: `0x10000` → `0x8000`
* 0697dbb on 2020-03-19: `0xF000` → `0x10000`
* 875c495 on 2020-03-19: `0x8000` → `0xF000`
* f7110c4 on 2015-09-17: `0x1000` → `0x8000`

Now in 2023, perhaps we should increase directly to 128 KB (`0x20000`) or even 256 KB (`0x40000`).